app/views/carts/show.html.haml

```diff
@@ -0,0 +1,3 @@
+
+= show
```

Comment:
What does this do? Don't misunderstand me, I don't mind helper methods, but I can't see youbhave defined it anywhere.

Reply:
Hmm I thought I had defined it in carts controller but maybe I'm getting helper methods wrong :/